README.md

```diff
@@ -88,9 +89,16 @@ drop and drag them into place or just select them with the chooser.
 Provide a title and description for the release and then click on the
 "Publish" button.
 
-For the documentation, log into ReadTheDocs.org and add the tag for the
-release.  Once added it will build automatically.  You can then select
-the latest release as the default version to show.
+After producing the release, the documentation on ReadTheDocs (RTD) must be updated,
+as RTD currently does not build automatically new versions.
+You must log into ReadTheDocs.org with an account who is
+listed as a maintainer for the `quattor-pan` project and add the tag for the
+release to the list of versions to build (this is done in the `Versions` tab
+of the `Settings` page access through the `Admin` button of the project home page).
+Once added it will build automatically.  The default version displayed on
+ReadTheDocs is normally `stable` which will automatically mapped to the new version
+once it is built (if it is the highest tag). You can also explicitly select the version
+to show on lower-left corner.
```

Comment:
This is a per-user-session setting though…

Reply:
No this is part of the project settings... A user can change this but the default is really ` stable`...

Reply:
Oh I see, you mean that the project admins can change which value appears there by default.